src/State/SearchResult/reducer.ts

```diff
@@ -0,0 +1,143 @@
+import initialState from '../../Reducers/initialState';
```

Comment:
file name should be reducers, not reducer just so we can be consistent with the other state files

Reply:
Idk why its showing single. on my code its plural